lms/djangoapps/bigcommerce_app/utils.py

```diff
@@ -342,6 +344,9 @@ def get_bc_course_enrollments(cls, user):
             enroll_courses = cls.get_order_items(bc_customer_id)
 
             if len(enroll_courses):
-                for course_key in enroll_courses:
-                    course_key = CourseKey.from_string(course_key)
-                    CourseEnrollment.enroll(user, course_key)
+                for course in enroll_courses:
+                    course_key = CourseKey.from_string(course['course_id'])
+                    if course['order_status'] == 'Completed':
+                        CourseEnrollment.enroll(user, course_key)
+                    elif course['order_status'] == 'Refunded' or course['order_status'] == 'Cancelled':
```

Comment:
@becdavid In https://jira.cymanii.inl.gov/browse/PB-50 we talked about unenrolling for the following statuses. Should we also include `Declined` and `Disputed` here?

> Refunded, Cancelled, Declined, Disputed
Unenroll the learner in the course

BigCommerce Orders API
https://developer.bigcommerce.com/api-reference/store-management/orders/order-status/getorderstatus

Reply:
I added the additional statuses. They will result in the user being unenrolled in the course. Any other order status apart from those listed will result in no action.